bin/tictactoe

```diff
@@ -1,3 +1,54 @@
 #!/usr/bin/env ruby
-
+require 'pry'
```

Comment:
You don't need to require pry, it's in the Gemfile and it's being loaded by `config/environment`